lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -222,6 +231,17 @@ def cert_action(module, iam, name, cpath, new_name, new_path, state,
             module.exit_json(changed=changed, msg='Certificate with the name %s already absent' % name)
 
 
+def load_data(cert, key, cert_chain):
+    # if paths are provided rather than lookups read the files and return the contents
+    if os.path.isfile(cert):
+        cert = open(cert, 'r').read().rstrip()
+    if os.path.isfile(key):
+        key = open(key, 'r').read().rstrip()
+    if os.path.isfile(cert_chain):
+        cert_chain = open(module.params.get('cert_chain'), 'r').read()
+    return cert, key, cert_chain
+
+
```

Comment:
Could probably get rid of the surplus required=False entries I seem to have missed as well